src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -21,12 +23,11 @@ trait EntityComponent {
     def workspaceId = column[UUID]("workspace_id")
     def workspace = foreignKey("FK_ENTITY_WORKSPACE", workspaceId, workspaceQuery)(_.id)
     def uniqueTypeName = index("idx_entity_type_name", (workspaceId, entityType, name), unique = true)
-
     def * = (id, name, entityType, workspaceId) <> (EntityRecord.tupled, EntityRecord.unapply)
   }
 
   class EntityAttributeTable(tag: Tag) extends Table[EntityAttributeRecord](tag, "ENTITY_ATTRIBUTE") {
-    def attributeId = column[Long]("attribute_id", O.PrimaryKey, O.AutoInc)
```

Comment:
note to all, this was a copy/paste bug that was a royal pain... it manifests itself as slick silently ignoring the values you tell it to insert